jcl/src/jdk.jcmd/share/classes/openj9/tools/attach/diagnostics/tools/Jstack.java

```diff
@@ -76,9 +73,9 @@ public static void main(String[] args) {
 			}
 			try {
 				diagProvider.attach(vmid);
-				DiagnosticsInfo groupInfo = diagProvider.getThreadGroupInfo(printSynchronizers);
-				out.printf("Virtual machine: %s JVM information %s%n", vmid, groupInfo.getJavaInfo()); //$NON-NLS-1$
-				out.println(groupInfo.toString());
+				out.printf("Virtual machine: %s JVM information:%n", vmid); //$NON-NLS-1$
+				Util.runCommandAndPrintResult(diagProvider, DiagnosticUtils.makeThreadPrintCommand(printSynchronizers), "jstack"); //$NON-NLS-1$
```

Comment:
This changes the way the information is presented to users: was that intentional? If so, why?

Reply:
Yes.  I think it makes it more readable:

Old:
```
$ jstack $!
2019-06-21T11:16:00.159960
Virtual machine: 90870 JVM information JRE 11 Mac OS X amd64-64-Bit Compressed References 20190618_000000 (JIT enabled, AOT enabled)
OpenJ9   - e92926a1b3
OMR      - 9b89971e
JCL      - ad988f03e0 based on jdk-11.0.4+6
"main" prio=5 Id=1 TIMED_WAITING
	at 11.0.4-internal/java.lang.Thread.sleep(Native Method)
	at 11.0.4-internal/java.lang.Thread.sleep(Thread.java:961)
	at app//looper.main(looper.java:18)
	CPU time: 120604000 ns
	User time: 120604000 ns
```
New:
```
$ jstack $!
2019-06-21T11:58:34.702411
Virtual machine: 44644 JVM information:
JRE 11 Mac OS X amd64-64-Bit Compressed References 20190621_000000 (JIT enabled, AOT enabled)
OpenJ9   - 3b21a2ed2c
OMR      - 52ffab05
JCL      - efbb0a226e based on jdk-11.0.4+8

"main" prio=5 Id=1 TIMED_WAITING
	at java.base@11.0.4-internal/java.lang.Thread.sleep(Native Method)
	at java.base@11.0.4-internal/java.lang.Thread.sleep(Thread.java:961)
	at app//looper.main(looper.java:18)
```